gce/image/build_image.sh

```diff
@@ -21,14 +21,10 @@ print_usage() {
     echo "build_image.sh --localrpm --repo [URL] --target [distribution]"
     echo "  --localrpm           deploy locally built rpms"
     echo "  --repo               repository for both install and update, specify .repo file URL"
-    echo "  --repo-for-install   repository for install, specify .repo file URL"
-    echo "  --repo-for-update    repository for update, specify .repo file URL"
     echo "  --product            scylla or scylla-enterprise"
-    echo "  --download-no-server download all rpms needed excluding scylla using .repo provided in --repo-for-install"
```

Comment:
Why removing this option ? This one is for who want build his custom version with his own scylla

Reply:
Can't they use --localrpm?
isn't it the same?

Reply:
This option help gather all the other rooms, to build only the parts you want to change

Reply:
I.e. see the README for the AMI for example of usage of it

Reply:
OK, i understand the different, let me fix it

Reply:
Fixed and tested locally